src/swarm-scenario/Swarm/Game/Entity.hs

```diff
@@ -436,7 +436,7 @@ data EntityMap = EntityMap
 instance Semigroup EntityMap where
```

Comment:
Should we use the term "right-biased" in the docs?

Reply:
Good idea, done.